lib/commands/slicemachine/commands/init.js

```diff
@@ -76,19 +76,19 @@ function generateProjectConfig(manifest, info) {
   }
 }
 
-function recap(manifest, info) {
+function handleRecap(manifest, recap, library, info) {
   clear();
 
   if (ctx.SliceMachine.isSetup) {
     generateProjectConfig(manifest, info);
-  } else {
-    const additionalDisplay = Mustache.render(manifest.recap, {
-      ...manifest,
-      ...info,
-    });
-
-    Helpers.UI.display(`\n\n${additionalDisplay}`);
   }
+  const additionalDisplay = Mustache.render(recap, {
+    ...manifest,
+    ...library,
```

Comment:
how it this used?

Reply:
the recap is created on server-side and might use any value present in the manifest, the fetched library or the info object

Reply:
oh yes ok